modules/siddhi-core/src/main/java/org/wso2/siddhi/core/util/persistence/FileSystemPersistenceStore.java

```diff
@@ -117,6 +118,28 @@ public String getLastRevision(String siddhiAppName) {
         return lastRevision;
     }
 
+    @Override
+    public void clearAllRevisions(String siddhiAppName) {
+        File targetDirectory = new File(folder + File.separator + siddhiAppName);
+        File[] files = targetDirectory.listFiles();
+
+        if (files == null || files.length == 0) {
+            log.info("No revisions were found with the Siddhi App " + siddhiAppName);
+            return;
+        }
+
+        for (File file : files) {
+            if (file.exists()) {
+                if (!file.delete()) {
+                    log.error("file is not deleted successfully!");
```

Comment:
Need file name here

Reply:
@ShashiLakshan You will need to add the file name to log message too